projs/shadow/shadow-engine/shadow-renderer/src/vulkan/VulkanModule.cpp

```diff
@@ -194,9 +194,12 @@ void VulkanModule::BeginRenderPass(const std::unique_ptr<vlkx::RenderCommand>& c
                             ImGui_ImplSDL2_NewFrame();
                             ImGui::NewFrame();
 
-                            if (editorEnabled)
-                                ImGui::Image(editorRenderPlanes[0], { 640, 480 });
-
+                            if (editorEnabled) {
+                                static bool active = true;
+                                ImGui::Begin("Game View", &active, ImGuiWindowFlags_None);
```

Comment:
If you don't wrap the ImGui::Image(...) in an if statement, this boolean parameter does nothing.

The intended behaviour is:
```
static bool active = true;
if (ImGui::Begin("Game View", &active, ImGuiWindowFlags_None)) {
     ImGui::Image((ImTextureID) editorRenderPlanes[0], {640, 480});
}
ImGui::End();
```

The boolean is just used by ImGui to keep track of the state of the window, it does nothing with it.